vllm/model_executor/weight_utils.py

```diff
@@ -19,6 +19,8 @@
                                                      QuantizationConfig)
 # UPSTREAM SYNC: needed for sparsity
 from vllm.model_executor.layers.parameters import LazyCompressedParameter
+from sparseml.transformers.compression.utils import (
```

Comment:
We're moving this to a separate repo right?

Reply:
yes! work in progress